cmd/klever-model-registry/main.go

```diff
@@ -15,27 +14,10 @@ import (
 	"github.com/kleveross/klever-model-registry/pkg/registry/modifiers"
 )
 
-const (
-	// kleverModelRegistryPort is model registry default port, default 8080
-	kleverModelRegistryPort = "klever_model_registry_port"
-)
-
-func init() {
-	viper.AutomaticEnv()
```

Comment:
Need  retain `viper.AutomaticEnv()` so that get other env.
If not call AutomaticEnv,  I am not sure can we get env value?

Reply:
Nice review, I will fix.